src/coreclr/jit/importercalls.cpp

```diff
@@ -4194,7 +4204,12 @@ void Compiler::impPopCallArgs(CORINFO_SIG_INFO* sig, GenTreeCall* call)
         }
         else
         {
+            // TODO-LLVM: lowerCallToShadowStack fails without the SignatureCorInfoType set (which is TARGET_WASM specific in NewCallArg 
+#if defined(TARGET_WASM)
+            arg = NewCallArg::Primitive(argNode, params[i - 1].CorType);
+#else
             arg = NewCallArg::Primitive(argNode, jitSigType);
+#endif
```

Comment:
```suggestion
            arg = NewCallArg::Primitive(argNode, params[i - 1].CorType);
```
Or did that not work?

Reply:
I didn't try it to be honest, but yes that does work and is now changed as per the suggestion, thanks.